extras/src/main/java/com/google/gson/typeadapters/UtcDateTypeAdapter.java

```diff
@@ -19,15 +19,12 @@
 import java.io.IOException;
 import java.text.ParseException;
 import java.text.ParsePosition;
-import java.util.Calendar;
-import java.util.Date;
-import java.util.GregorianCalendar;
-import java.util.Locale;
-import java.util.TimeZone;
+import java.util.*;
```

Comment:
Wildcard imports [should be avoided](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports)

Reply:
+1